marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/models/getPrimaryEntityTypes.sjs

```diff
@@ -26,7 +27,7 @@ let modelResponseArr = fn.collection(entityLib.getModelCollection()).toArray().m
   const response = {
     entityName: entityName,
     entityTypeId: entityLib.getEntityTypeId(model, entityName),
-    entityInstanceCount: cts.estimate(cts.collectionQuery(entityName)),
+    entityInstanceCount: cts.estimate(ext.buildEntityQuery(entityName)),
```

Comment:
@rahulvudutala  Any idea if there are other places in the codebase where DHF is assuming that it can do a collection query on the entityName?

Reply:
@rjrudin There is mastering but that will, unfortunately, be more difficult to patch due to it being in XQuery.

Reply:
Yeah that's fine for now - I should have qualified my question by asking @rahulvudutala  if there was another Explore-related place.